README.md

```diff
@@ -12,662 +12,24 @@
 MLRun is an open-source MLOps framework that offers an integrative approach to managing your machine-learning pipelines from early development through model development to full pipeline deployment in production.
 MLRun offers a convenient abstraction layer to a wide variety of technology stacks while empowering data engineers and data scientists to define the feature and models.
 
```

Comment:
@jillnogold i suggest to add the architecture diagram and links to Katacoda and mlrun.org, and can embed the tutorial video (https://www.youtube.com/watch?v=O6g1pJJ609U)

Reply:
done